static/docs/command-reference/version.md

```diff
@@ -20,6 +20,7 @@ system/environment:
 | `Python version`                            | Version of the Python being used on the environment in which DVC is initialized                                                                                       |
 | `Platform`                                  | Information about the operating system of the machine                                                                                                                 |
 | [`Binary`](#what-we-mean-by-binary)         | Shows whether DVC was installed from a package or from a binary release                                                                                               |
+| `Package manager`                           | Shows the package manager name that was used to install DVC (`pip`, `conda`, etc)                                                                                     |
```

Comment:
How about "If not `Binary`, name of the package manager used to install DVC (e.g. `pip`, `conda`, etc.)"

Reply:
it's not true unfortunately, there are Binary package managers. Yes, it is confusing and probably should be improved on the DVC core side first - I would integrate and use a single field - method of install, instead of this package manager + binary.

Reply:
OK. Still i would rephrase to just "Name of the package manager used to install DVC if any (e.g. `pip`, `conda`, etc.)"

Reply:
done! thanks, sounds way better